plugins/reference-assets/android/deps.bzl

```diff
@@ -1,20 +1,17 @@
 load("//jvm/dependencies:versions.bzl", "versions")
 load("@rules_player//maven:parse_coordinates.bzl", "parse_coordinates")
+load("//plugins/reference-assets/android/src/androidTest/java/com/intuit/player/android/reference/assets/test:deps.bzl", maven_test = "maven")
 
-maven = [
-    "org.jetbrains.kotlinx:kotlinx-coroutines-android:%s" % versions.kotlin.coroutines,
-
-    "androidx.appcompat:appcompat:1.2.0",
-    "androidx.core:core-ktx:1.3.2",
-    "androidx.constraintlayout:constraintlayout:2.0.4",
-]
+maven_main = []
 
 main_exports = [
     "//android/player",
 ]
 
-main_deps = main_exports + parse_coordinates(maven) + [
+main_deps = main_exports + parse_coordinates(maven_main) + [
```

Comment:
since `maven_main` doesn't have anything in it right now, maybe remove the `parse_coordinates`, i'm not against keeping the `maven_main` though

Reply:
This is more for the patternization of `deps.bzl`. If someone wants to add another dependency, all they need to do right now is add `"androidx.core:core-ktx:%s" % versions.androidx.core" and that will automatically be pulled into the `maven_install` _and_ converted to the corresponding `@maven//:androidx_core_core_ktx` Bazel reference.

If we remove the `parse_coordinates`, then they'd need to do:
> converted to the corresponding `@maven//:androidx_core_core_ktx` Bazel reference.
manually, or reintroduce the `parse_coordinates` method.